draft-ietf-quic-transport.md

```diff
@@ -1691,13 +1722,13 @@ waiting for a WINDOW_UPDATE which will never come.
 ### Mid-stream RST_STREAM
 
 On receipt of an RST_STREAM frame, an endpoint will tear down state for the
```

Comment:
for the what?

I think that you can:

1. free any buffers allocated to incoming data
2. discard any data that hasn't been delivered yet

But you have to:

1. account for the octets consumed from the connection flow control window (those that aren't already accounted for)
2. keep the octet offset so that you can reject overflows
3. keep the state for outgoing data in place